### PR TITLE
Introduction to PyDP added to readme.md(refactored)

### PR DESCRIPTION
**Pull request by ghost**
## Description
Please include a summary of the change, the motivation, and any additionl context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,76 @@
...
 
 This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, PyDP is helping us achieve better privacy.
 
+**Brief introduction to google's Differential privacy project :-** 
+
+The Differentia Privacy project of google contains a set of libraries of ε- and (ε, δ)-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. The functionality is currently available in C++, Go and Java.
+
+Currently, it provides algorithms to compute the following:
```
Comment (chinmayshah99) [inline README.md:15]: Can we remove this table?

File: README.md [hunk 2]
```diff
@@ -2,30 +2,76 @@
...
+| Mean               | Supported     | Supported |Supported |
+| Variance           | Supported     | Planned   |Planned   |
+| Standard deviation | Supported     | Planned   |Planned   |
+| Order statistics (incl. min, max, and median) | Supported   | Planned | Planned |
+| Automatic bounds approximation | Supported   | Planned | Planned |
+
+It also provides implementations of the Laplacian and Gaussian Mechanisms that can be used to perform computations that aren't covered by the pre-built algorithms.
+All of these algorithms are suitable for research, experimental or production use cases.
+
+This project also contains:
```
Comment (chinmayshah99) [inline README.md:30]: Even this section needs to removed

File: README.md [hunk 3]
```diff
@@ -2,30 +2,76 @@
...
+| -------------      |:-------------:|:---------:|:--------:|
+| Count              | Supported     | Supported |Supported |
+| Sum                | Supported     | Supported |Supported |
+| Mean               | Supported     | Supported |Supported |
+| Variance           | Supported     | Planned   |Planned   |
+| Standard deviation | Supported     | Planned   |Planned   |
+| Order statistics (incl. min, max, and median) | Supported   | Planned | Planned |
+| Automatic bounds approximation | Supported   | Planned | Planned |
+
+It also provides implementations of the Laplacian and Gaussian Mechanisms that can be used to perform computations that aren't covered by the pre-built algorithms.
```
Comment (chinmayshah99) [inline README.md:27]: L#27and #28 also needs to be removed

File: README.md [hunk 4]
```diff
@@ -2,30 +2,76 @@
...
+More information [here](https://github.com/google/differential-privacy)
+
+
 **Things to remember about PyDP :**
 - :rocket: Features differentially private algorithms including: BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median, etc.  
 - All the computation methods mentioned above use Laplace noise only. (Other noise mechanisms will be added soon... :smiley:)
 - :fire: Currently supports Linux and OSX. (Windows coming real soon... :smiley:)
 - :star: Supports all the Python 3+ versions.
 
+**Key features of PyDP :-**
```
Comment (chinmayshah99) [inline README.md:48]: How is this different from the above section?
This just seems duplication

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

**Comment by chinmayshah99** [comment]
This readme is supposed to be in the /docs folder.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:15]: Yes we can remove it, I added it to add a brief intro to google's DP project.

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:30]: sure!

**Pull request closed without merging**